music21/stream/base.py

```diff
@@ -10174,22 +10182,35 @@ def pitches(self) -> list[pitch.Pitch]:
         for e in self.elements:
             if isinstance(e, key.Key):
                 continue  # has .pitches but should not be added
-            elif isinstance(e, Stream):
-                # Stream has a pitches properties; this just
-                # causes a recursive pitch gathering
+            # both GeneralNotes and Stream have a pitches properties; this just
+            # causes a recursive pitch gathering
+            elif isinstance(e, (note.GeneralNote, Stream)):
                 post.extend(list(e.pitches))
-            elif isinstance(e, note.GeneralNote):
-                # GeneralNote is ultimately where we get all the pitches
-                if e.pitches:
-                    post.extend(list(e.pitches))
-                    # and also any of the GeneralNote's ornaments' otherPitches
-                    for orn in e.expressions:
-                        if isinstance(orn, expressions.Ornament):
-                            # Figure out what the other pitches are, in case that hasn't
-                            # been done yet (this is a no-op if it has already been done).
-                            orn.resolveOtherPitches(e.pitches[-1])
-                            # Add them to the list
-                            post.extend(list(orn.otherPitches))
+        return post
+
+    @property
+    def ornamentalPitches(self) -> list[pitch.Pitch]:
```

Comment:
remove from stream -- it can be in expressions.py, etc.  see above.  Unlikely to be used by most users and reducing music21 complexity is a goal.

Reply:
Or since it's only used in makeNotation it can I suppose live in makeNotation.py